.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Prebuild
+
+jobs:
+  prebuild-node:
+    name: Upload Node Prebuild
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
I don't see a property called `node-version` in the matrix defined above. Is this a typo?

Reply:
Yes, the `matrix.node-version` in the name is a residue form a previous version of config I wrote. that can be removed from the name. with prebuild, we can build for other node versions from the same LTS version that we use.